### PR TITLE
Add low Gnss laikad test

### PR DESCRIPTION
**Pull request by gast04**

### CODE REVIEW

**Comment by gast04** [comment]
merge after https://github.com/commaai/laika/tree/fix_trop_delay

File: selfdrive/locationd/laikad.py [hunk 1]
```diff
@@ -391,13 +391,16 @@ def process_msg(laikad, gnss_msg, mono_time, block=False):
 
 
 def clear_tmp_cache():
+  if "CI" in os.environ or "REPLAY" in os.environ:
+    return
```
Comment (adeebshihadeh) [inline selfdrive/locationd/laikad.py:395]: why does CI rely on this?

Comment (gast04) [reply selfdrive/locationd/laikad.py:395]: replay uses CI env flag, and runs it in parallel, thats doesnt work

Comment (adeebshihadeh) [reply selfdrive/locationd/laikad.py:395]: hmm, can you add a comment to remove this once we're not writing this cache to disk? CI exceptions should be used extremely sparingly and only when absolutely necessary

**Pull request merged**